client/my-sites/stats/features/modules/stats-locations/stats-locations.tsx

```diff
@@ -30,9 +30,9 @@ const OPTION_KEYS = {
 	CITIES: 'cities',
 };
 
-type StatQueryType = 'country' | 'region' | 'city';
+type GeoMode = 'country' | 'region' | 'city';
```

Comment:
I changed the name to avoid confusion between `statType` and `statQueryType`. Geomode makes a bit more sense and it's the name of the argument of `useLocationViewsQuery()`.

Reply:
Makes sense :)